Replicator/Pusher.cc

```diff
@@ -684,7 +776,7 @@ namespace litecore { namespace repl {
             level = kC4Stopped;
         } else if (isBusy()) {
             level = kC4Busy;
-        } else if (_options.push == kC4Continuous || isOpenServer()) {
+        } else if (_continuous || isOpenServer() || !_conflictsIMightRetry.empty()) {
```

Comment:
@snej  ^

Reply:
The _Pusher_ can go idle, when it's got a potential conflict but is waiting to see if the Puller will update the remote revID. 

The Replicator won't be confused by this; if it's one-shot and the subcomponents are idle, it will stop (see Replicator.cc:251 or thereabouts.)